app/scripts/migrations/120.1.ts

```diff
@@ -0,0 +1,48 @@
+import { cloneDeep, isObject } from 'lodash';
+import { hasProperty } from '@metamask/utils';
+
+type VersionedData = {
+  meta: { version: number };
+  data: Record<string, unknown>;
+};
+
+export const version = 120.1;
+
+/**
+ * Add a default value for importTime in the InternalAccount
+ *
+ * @param originalVersionedData
+ */
+export async function migrate(
+  originalVersionedData: VersionedData,
+): Promise<VersionedData> {
+  const versionedData = cloneDeep(originalVersionedData);
+  versionedData.meta.version = version;
+  transformState(versionedData.data);
+  return versionedData;
+}
+
+function transformState(
+  state: Record<string, unknown>,
+): Record<string, unknown> {
+  // Existing users who do not have UserStorageController state.
+  // Provide some initial state & nullify `isProfileSyncingEnabled`
+  if (!hasProperty(state, 'UserStorageController')) {
+    state.UserStorageController = {
+      isProfileSyncingEnabled: null,
+    };
+    return state;
+  }
+
+  // Existing users who do have UserStorageController state.
+  // nullify `isProfileSyncingEnabled`
+  if (
+    isObject(state.UserStorageController) &&
+    hasProperty(state.UserStorageController, 'isProfileSyncingEnabled')
+  ) {
+    state.UserStorageController.isProfileSyncingEnabled = null;
+    return state;
+  }
```

Comment:
Nit: It looks like we have an uncovered branch here, the case where the UserStorageController state is invalid. We should handle the case where the `UserStorageController` is set incorrectly (e.g. the state was corrupted).

Usually in cases like this, if it's totally unfathomable how it could be set to anything different, we would submit a Sentry error. Note that this would require importing `captureException` and adding a new test as well.

```suggestion
  if (
    !isObject(state.UserStorageController) ||
    !hasProperty(state.UserStorageController, 'isProfileSyncingEnabled')
  ) {
    captureException(
      `Migration ${version}: Invalid UserStorageController state: ${typeof state.PreferencesController}`,
    );
    return state;
  }

  // Existing users who do have UserStorageController state.
  // nullify `isProfileSyncingEnabled`
  state.UserStorageController.isProfileSyncingEnabled = null;
```

Reply:
Oooooh yep ty for raising this. Yeah I'll add some capture exceptions in case for corrupted state. Will add tests too!